src/utils/sceneimagebuilder/sceneimagebuilder.cpp

```diff
@@ -3,47 +3,27 @@
 // Purpose: 
 //
 //=====================================================================================//
-#include "appframework/AppFramework.h"
-#include "tier1/strtools.h"
-#include "tier0/icommandline.h"
-#include "tools_minidump.h"
-#include "filesystem_init.h"
-#include "filesystem_tools.h"
-#include "filesystem.h"
-#include "loadcmdline.h"
-#include "scriplib.h"
-#include "utlbuffer.h"
-#include "cmdlib.h"
-#include "sceneimage.h"
-#include "choreoevent.h"
+#include <appframework/AppFramework.h>
```

Comment:
<> causes compiler to search in system header paths first, local path second which can extend compilation time.